src/mmw/apps/modeling/serializers.py

```diff
@@ -195,6 +198,25 @@ def create(self, validated_data):
         if 'user' not in validated_data:
             validated_data['user'] = self.context['request'].user
 
+        if validated_data['model_package'] == Project.GWLFE and (
+           'weather_simulations' not in validated_data or
+           validated_data['weather_simulations'] == []):
+            # Initialize weather simulations for this project
+            try:
+                validated_data['weather_simulations'] = \
+                    get_available_simulations_for_aoi(
+                        validated_data['area_of_interest'])
```

Comment:
Since this makes a call to S3 to check eligibility, that's an external dependency that could potentially fail. Currently we wrap this in a `try` and `except` to the default value (all simulations disabled), to allow for a project to still be created. However, this behavior may be mysterious to users, who won't know why their project isn't eligible for certain simulations.